features/step_definitions/skin_steps.rb

```diff
@@ -182,6 +182,10 @@
 
 ### THEN
 
+Then /^I should see a parent skin text field$/ do
```

Comment:
Can we update the syntax here to `Then "I should see a parent skin text field" do`?